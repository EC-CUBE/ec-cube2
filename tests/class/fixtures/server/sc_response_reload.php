<?php

require __DIR__.'/common.php';

$_SERVER['REQUEST_URI'] = HTTPS_URL.'index.php?debug='.urlencode('テスト');
$_SESSION[TRANSACTION_ID_NAME] = 'aaaa';

SC_Response_Ex::reload(['redirect' => 1]);
