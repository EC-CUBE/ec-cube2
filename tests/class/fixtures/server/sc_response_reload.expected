
Array
(
    [0] => Content-Type: text/plain; charset=utf-8
    [1] => Location: http://127.0.0.1:8085/index.php?debug=%E3%83%86%E3%82%B9%E3%83%88&redirect=1
)
shutdown
